coalib/bearlib/languages/documentation/DocumentationExtraction.py

```diff
@@ -1,3 +1,10 @@
+"""
+Language and docstyle independent documentation comments extraction.
+
+Each of the functions is built upon one another, and at the last,
+exposes a single function py:function:`extract_documentation_with_markers`
```

Comment:
you do not need to do it manually, coala has helper commands :)

Just do `pip install -r docs-requirements.txt` then `./setup.py docs`, that should build the docs